docs/index.md

```diff
@@ -1,71 +1,126 @@
+
+Public declarations of potential conflicts of interest are a required component of the checks and balances in a democratic society. When these declarations only have value in isolation and cannot easily be compared with others, or linked to other types of information, they have limited value. Especially so at a time when private sector and non-democratic actors face no such limitations.
+
+If democratic institutions and bodies adopted an open data standard for publishing their registers of interests, tools such as [declared.info](http://declared.info/) could be built cheaply and have far-reaching scope. The public interest would be far better served.
+
+## Status of the data model
+
+We present here the beginnings of an open data standard: a draft data model to represent information declared by people about their interests. The structure of the data (the data model) is informed by:
+
+* the experience of dealing with unstructured, variously formatted information in this domain,
+* an understanding of how tightly the format of declarations can be tied to regulatory requirements and language,
+* a recognition that an 'interest' may cover an extremely wide range of activities, roles and engagements,
+* an assumption that the added value of a structured format is in enabling patterns across declarations to be more easily identified,
+* experience elsewhere in the transparency field which highlights the importance of historical data on declarations.
+
+The data model is a first draft: it's an opener in a conversation. Publishing its details here allows the underlying ideas to feed into further work in this domain. We hope that it might provoke interest and criticism.
+
+The [draft data model](https://docs.google.com/spreadsheets/d/1QCVkxi1B-i3xx1lVJXe1ihhQSrgtOEeAxj8CHoFgTVA/edit#gid=0) is documented in a spreadsheet for convenience. Ultimately a related data standard could be rendered in JSON schema.
+
+## Overview of the data model
 
-## Examples
+![Diagram of data model](_assets/UML-interests-data-model.png)
 
-Table without horizonal scrollbar:
+The data model encapsulates an Interest object, placing it within a Declaration. This allows processes of declaration and re-declaration (or confirmation) to be treated and understood separately from the life-cycle and nature of the interest itself. (For example, a person may be required to declare an interest which actually sits with a family member or business partner. And they may confirm that the same interest exists in a future declaration.)
```

Comment:
same interest *still* exists

Reply:
I don't think that the parentheses add anything here; removing them from the second sentence improves readability.

Reply:
Agreed.